src/main/java/com/rarchives/ripme/ui/MainWindow.java

```diff
@@ -1238,7 +1238,8 @@ public void actionPerformed(ActionEvent event) {
                                 queueListModel.add(queueListModel.size(), realURL);
                                 ripTextfield.setText("");
                             } else {
-                                LOGGER.error("Can't find ripper for " +realURL);
+                                appendLog("Can't find ripper for " + realURL, Color.RED);
+                                LOGGER.error("Can't find ripper for " + realURL);
```

Comment:
This seems like a combination we probably do a lot and might want to refactor to a function to take the message only one time and do both (related) actions?